examples/dt/nova/nova-three-cells/control-plane/service-values.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  preserveJobs: false
+  galera:
+    templates:
+      openstack:
+        storageClass: lvms-local-storage
+        storageRequest: 5G
+        secret: osp-secret
+        replicas: 3
+      openstack-cell1:
+        storageClass: lvms-local-storage
+        storageRequest: 5G
+        secret: osp-secret
+        replicas: 3
+      openstack-cell2:
+        storageClass: lvms-local-storage
+        storageRequest: 5G
+        secret: osp-secret
+        replicas: 3
+
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:swift
+      [glance_store]
+      default_backend = default_backend
+      [default_backend]
+      swift_store_create_container_on_put = True
+      swift_store_auth_version = 3
+      swift_store_auth_address = {{ .KeystoneInternalURL }}
+      swift_store_endpoint_type = internalURL
+      swift_store_user = service:glance
+      swift_store_key = {{ .ServicePassword }}
+    default:
+      replicas: 1
+
+  swift:
+    enabled: true
+  telemetry:
+    enabled: true
+    template:
+      ceilometer:
+        enabled: true
+
+  rabbitmq:
+    templates:
+      rabbitmq-cell2:
+        replicas: 3
```

Comment:
Is this intentional that for galera above all the 3 DBs are re-defined, but for rabbitmq here only the additional MQ is defined?

Reply:
Please use a separate MQ cluster for each cell

Reply:
@gibizer I need to take a look at this again, you actually helped me debug this back in early October.  The deployment was requesting something like 500M by default so I updated all of them to 5G.  Although looking at the architecture library it should be using 5G.  Let me revisist why this was happening and I'll reduce if possible.

@bogdando ack, thanks.

Reply:
@bogdando When you say separate MQ for each cluster do you mean unique metallb annotations per rabbitmq cluster?  If so I'm currently getting those via CIFMW.  I can add that as part of the README to help clarify.

```
  rabbitmq:
    enabled: true
    templates:
      rabbitmq:
        delayStartSeconds: 30
        override:
          service:
            metadata:
              annotations:
                metallb.universe.tf/address-pool: internalapi
                metallb.universe.tf/loadBalancerIPs: 172.17.0.85
...
      rabbitmq-cell1:
        delayStartSeconds: 30
        override:
          service:
            metadata:
              annotations:
                metallb.universe.tf/address-pool: internalapi
                metallb.universe.tf/loadBalancerIPs: 172.17.0.86
...
      rabbitmq-cell2:
        delayStartSeconds: 30
        override:
          service:
            metadata:
              annotations:
                metallb.universe.tf/address-pool: internalapi
                metallb.universe.tf/loadBalancerIPs: 172.17.0.87
```

Reply:
I think the current proposal already uses individual rabbit for each cell. 
cell0: rabbit-server
cell1: rabbit-cell1-server
cell2: rabbit-cell2-server
So I think we are good.

Reply:
Ack I'll remove the 3 galera and keep it inline with how rabbitmq is being defined here.